injector/src/main/java/edu/ucr/cs/riple/injector/offsets/FileOffsetStore.java

```diff
@@ -113,9 +114,11 @@ public Path getPath() {
    * @param existingOffsetChanges List of existing offset changes.
    * @return Translated and sorted offsets.
    */
-  public List<OffsetChange> getOffsetsRelativeTo(List<OffsetChange> existingOffsetChanges) {
+  public SortedSet<OffsetChange> getOffsetWithoutChanges(
+      SortedSet<OffsetChange> existingOffsetChanges) {
     return offsetChanges.stream()
         .map(offsetChange -> offsetChange.getOffsetWithoutChanges(existingOffsetChanges))
-        .collect(Collectors.toList());
+        .sorted(comparingInt(o -> o.position))
```

Comment:
Why is the explicit sorting needed here if we are using a sorted data structure? What am I missing?

Reply:
This method is deleted in the latest changes (a11ed6f19d319dd07fa2e57160ed0f5576234fc2) and yes, it was not necessary and deleted it.